### ISSUE TITLE
Incorrect markup for "Heidi Van Pelt"

### ISSUE THREAD

**Issue by turkanis** [datamade/probablepeople#48]
The name "Heidi Van Melt" is marked up [here](https://github.com/datamade/probablepeople/blob/560691e55dd995d30fd23662e7be0864fb49bc62/name_data/labeled/person_labeled.xml#L3018) as 
`<Name><GivenName>Heidi</GivenName> <And>Van</And> <Surname>Pelt</Surname></Name>`
This appears to be incorrect. I would suggest:
`<Name><GivenName>Heidi</GivenName> <Surname>Van Pelt</Surname></Name>`
or
`<Name><GivenName>Heidi</GivenName> <MiddleName>Van</MiddleName> <Surname>Pelt</Surname></Name>`

**Comment by jeancochrane** [comment]
I agree with the `GivenName Surname Surname` tagging. Want to make a PR @turkanis?
